pkg/fanal/analyzer/os/ubuntu/ubuntu.go

```diff
@@ -42,12 +53,25 @@ func (a ubuntuOSAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInpu
 				},
 			}, nil
 		}
+
+		if input.FilePath == esmConfFilePath { // Check esm config file
```

Comment:
We may want to have another analyzer for ESM.

Reply:
Yes, it is better. Created new analyzer.